CHANGES.rst

```diff
@@ -1,27 +1,27 @@
 Changes
 =======
 
-Version 0.8.0 (UNRELEASED)
+Version 0.8.0 (2021-11-22)
 --------------------------
 
 - Adds new command ``quota-show``.
-- Adds possibility to filter by workflow status and search by workflow name to ``list`` command.
+- Adds new command ``info`` that retrieves general information about the cluster, including the list all the available workspace paths.
+- Adds possibility to filter by workflow status and search by workflow name to ``list`` command via ``--filter`` option.
+- Adds possibility to filter and display all the runs of a given workflow to the ``list`` command via ``-w`` option.
+- Adds possibility to filter by file name, size and last modified values to ``ls`` command via ``--filter`` option.
+- Adds possibility to filter by file name and size to ``du`` command via ``--filter`` option.
 - Adds job start and finish times to ``logs`` command output.
```

Comment:
Ditto here, perhaps "Changes `logs` command to add start and finish...".  Basically, the structure of the news could be "adds/changes/fixes command foo", and then explaining how.  This will allow to sort release notes news items by command, so  that users will find more easily what's new.